File: Uber/Java/uberX.java
class Uberx extends Car{
    String brand;
    String model;

    public Uberx(String license, Account driver, String brand, String model){
        super(license, driver); //Llamar a la clase padre(metodo constructor) "Car"
        this.brand=brand; //Asignar a la clase hija "la actual"
        this.model=model;
    }
    void printDataCar() {
        super.printDataCar();
        System.out.println("Modelo: " + model + " Brand: " + brand);
    }
}

File: Uber/PHP/uberX.php
<?php
require_once('car.php');
class   UberX extends Car {
    public $brand;
    public $model;

    public function __construct($license, $driver,$brand, $model) {
    parent::__construct($license,$driver);//Llama a la clase padre volviendola una instancia del objeto***
    $this->brand = $brand;
    $this->model = $model;
    }
    public function setPassenger($passenger) {
        if ($passenger == 4) {
            $this->passenger = $passenger;
        }
        else {
            echo "Necesitas asignar 4 pasajeros
    ";
        }
    }
    public function printDataCar() {
        echo "
            Licencia: $this->license
            Driver: {$this->driver->name}
            Número de pasajeros: $this->passenger
            Model:$this->model
            Bran:$this->brand
        ";
      }

}